mos-platform/cpm65/cpm.h

```diff
@@ -0,0 +1,183 @@
+#ifndef CPM_H
+#define CPM_H
+
```

Comment:
Add the C++ "extern C" headers so this header can work with C++ too. See various other headers in the SDK.

Reply:
Done.